conf/defaults.ini

```
@@ -1010,3 +1010,10 @@ default_timezone = browser
 [expressions]
 # Enable or disable the expressions functionality.
 enabled = true
+
+[geomap]
+# Set the default base layer in GeoMaps plugin
+geomap_default_baselayer =
```

if we add a [geomap] section, I don't think we need the geomap prefix for everything

```suggestion
default_baselayer =
```